mitmproxy/options.py

```diff
@@ -30,19 +30,19 @@ def __init__(
             app_port: int = APP_PORT,
             anticache: bool = False,
             anticomp: bool = False,
-            client_replay: Sequence[str] = (),
+            client_replay: Sequence[str] = [],
```

Comment:
Alternately, the usual idiom is something like this:

```python
client_replay: Sequence[str] = None,
```

Then in the body of the function:

```python
if not client_replay:
    client_replay = []
```